tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MpMetricTest.java

```diff
@@ -88,6 +89,9 @@
 
     private static final String APPLICATION_JSON = "application/json";
     private static final String TEXT_PLAIN = "text/plain";
+    
+    private static final String JSON_APP_LABEL_REGEX = "_app=[A-Za-z0-9]*;"; 
```

Comment:
wouldn't this fail to match if the `_app` tag is listed as the last one (or the only one), because then it will not be followed by a semicolon? I'd say the semicolon should be at the start of the regex, not the end. The end should be determined by either a semicolon (if there are more tags after this), or a quote (if there are no more tags).